README.md

```diff
@@ -73,7 +73,7 @@ To restore, we:
 
 - get the __entropy__ from the __mnemonic__
 - the __entropy__ is effectively the key __seed__, so we use it to create a SSH key pair
-- the key is effectively the same that was backup up, as the key is the same.
+- the key is effectively the same that was backed up, as the key is the same.
```

Comment:
Typo here?

Reply:
I think it was typo before, `backup up` sounds weird...